Dockerfile

```diff
@@ -8,11 +8,17 @@ ENV OVOS_CONFIG_FILENAME=diana.yaml
 ENV OVOS_DEFAULT_CONFIG=/opt/neon/diana.yaml
 ENV XDG_CONFIG_HOME=/config
 ENV CHATBOT_VERSION=v2
+ENV HEALTHCHECK_PORT=8000
```

Comment:
8080 to match default in `healthcheck.sh`

Reply:
On second thought, standardize around port 8000 as that is what k8s deployments were tested with and it makes no difference for Docker deployments since this is all internal